README.md

```diff
@@ -140,13 +145,12 @@ Winter CMS is also compatible with most OctoberCMS plugins available at https://
 - [Mahony0.Floc](https://packagist.org/packages/mahony0/wn-floc-plugin) - Provides middleware to disable FLoC.
 - [Mahony0.Updater](https://packagist.org/packages/mahony0/wn-updater-plugin) - Update WinterCMS via remote command.
 - [MarcoMessa.Vite](https://packagist.org/packages/marcomessa/wn-vite-plugin) - Plugin required for Vite TailwindCss Theme.
-- [Mercator.TwigExt](https://packagist.org/packages/mercator/wn-twigext-plugin) - Various Twig extensions.
+- [Mercator.TwigExt](https://packagist.org/packages/mercator/wn-twigext-plugin) - Various Twig extensions, based on OctoberCMS'[Twig Extensions](https://github.com/vojtasvoboda/oc-twigextensions-plugin) by Vojta Svoboda.
```

Comment:
This is the one that should be removed, plugins are supposed to be listed alphabetically here

Reply:
fixed!